rosetta-source/src/main/rosetta/product-common-settlement-type.rosetta

```diff
@@ -69,6 +70,13 @@ type PayoutBase: <" Base class that all payout types should extend. Use case is
     principalPayment PrincipalPayments (0..1) <"The specification of the principal exchange. Optional as only applicable in the case of cross-currency or zero-coupon swaps with a final payment.">
     settlementTerms SettlementTerms (0..1) <"Each payout leg must specifies its settlement terms, including the delivery type (i.e. cash vs physical, and their respective terms), the transfer type (DvP etc.) and settlement date, if any.">
 
+type CumulationFeature: <"Describe provisions where the initial notional quantity is incremented as per Cumulation Dates, which observations in period are either 'guarenteed' or submitted to Knock-In and/or Knock-Out conditions that would trigger or stop the quantity increment cumulation for the related Cumlulation Dates. Besides, some period may include 'lever' i.e. multiplier value to apply on the quantity increment as part of the given Cumulation Dates period.">
```

Comment:
do not agree - repeating a prefix may clarify what is at stake in some cases

- following your suggestion instead of "cumulationDates" we would have "date"... that is just quite "ugly" i cannot imagine any GUI that would not need to over-code it to produce a custom label "Cumulation Dates" etc. 
- besides, there are SO MANY occurrences of this way of drafting in CDM that at least, this shall not be "blocking point" for this particular PR i mean this is not a design principel or if it is let's refactor now thousands of names...

Reply:
The purpose of the CDM is to provide a logical model, not to fit the requirements of GUI implementors.

It is true that there are instances where prefixing an attribute name can provide welcome clarity - e.g. `tradeDate` in `Trade`. But to be effective, prefixing should be used wisely. Here, you have prefixed **all** the `CumulationFeature` attributes with 'cumulation', which looks excessive and unnecessary.

Reply:
maybe the word "GUI" is to far from CDM per se, but please consider that being "logical model" cannot mean to to opposite not to take into consideration potential gap between backend and forntend implementation, i mean as far as possible, backend shall try to faciliate, say take into account frontend, because we are not talking about "abstract logical model" : CDM is not Python or Java, it is based on Domain Languahge, i mean it is "business logical model" hence the "name" of object is really important and must be assessed in regards of business meaning - if not, to the extreme, imagine full ignorance of frontend, then CDM articfact would never be a standard and implementors would custom all "names" of objects, etc. 

all in all the point is in the middle of the bridge, i mean finding right balance between two extremes neither one being acceptable : "pure logical model" <> "pure business-GUI oriented" - hope you would agree when i re-phrase it that way ?